test/unit/presenters/publishing_api/take_part_presenter_test.rb

```diff
@@ -45,6 +45,8 @@ def present(record)
     assert_equivalent_html expected_hash[:details].delete(:body),
       presented_content[:details].delete(:body)
 
+    assert_equal expected_hash[:details][:image][:url], presented_content[:details][:image][:url]
```

Comment:
Do we need this new assertion given that the line below checks that `expected_hash` equals `presented_content`?

Reply:
No. Removed in fixup.